source/extensions/load_balancing_policies/maglev/maglev_lb.cc

```diff
@@ -81,16 +81,30 @@ void MaglevTable::constructMaglevTableInternal(
     return;
   }
 
-  // Implementation of pseudocode listing 1 in the paper (see header file for more info).
-  std::vector<TableBuildEntry> table_build_entries;
-  table_build_entries.reserve(normalized_host_weights.size());
+  // Prepare stable (sorted) vector of host_weight
```

Comment:
Can this also include why we need a stable (sorted) vector of host weights ? Since this was missed before and is non-obvious.

Reply:
Updated the comment https://github.com/envoyproxy/envoy/pull/28055/commits/17a4f8ae7a06a4fb672efba48c1c24db31dfb997